inc/redirect/namespace.php

```diff
@@ -198,7 +198,11 @@ function rewrite_rules_for_sitemap() {
 function rewrite_rules_for_open() {
 
 	add_rewrite_endpoint( 'open', EP_ROOT );
-	add_filter( 'template_redirect', __NAMESPACE__ . '\do_open', 0 );
+	add_filter( 'template_redirect', function () {
+		do_open( function ( $filepath ) {
+			force_download( $filepath );
+		} );
+	}, 0 );
```

Comment:
This was required to "mock" the download functionality. We now pass a callback to the `do_open` function and it will call it passing the file to be downloaded.